src/app/components/file-ownership-modal/file-ownership-modal.component.ts

```diff
@@ -0,0 +1,129 @@
+
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+  
+  SPDX-License-Identifier: EPL-2.0
+  
+  Copyright Contributors to the Zowe Project.
+*/
+import { Component, Inject } from '@angular/core';
+import { MAT_DIALOG_DATA, MatSnackBar } from '@angular/material';
+import { Observable } from 'rxjs/Observable';
+import { Http } from '@angular/http';
+
+@Component({
+  selector: 'file-ownership-modal',
+  templateUrl: './file-ownership-modal.component.html',
+  styleUrls: ['./file-ownership-modal.component.scss',
+    '../../../../src/app/shared/modal.component.scss'],
+})
+export class FileOwnershipModal {
+
+  public name = '';
+  public path = '';
+  public mode = 0;
+  public modeSym = '';
+  public icon = '';
+  public ownerName = "";
+  public groupName = "";
+
+  constructor(
+    @Inject(MAT_DIALOG_DATA) data,
+    private http: Http,
+    private snackBar: MatSnackBar,
+  ) 
+  {
+    const node = data.event;
+    this.name = node.name;
+    this.path = node.path;
+    this.mode = node.mode;
```

Comment:
This dialog doesn't show the current owners when I open it.

Reply:
![image](https://user-images.githubusercontent.com/34669449/92861629-2a552300-f413-11ea-9961-14ef881bda18.png)

Reply:
Yes agreed, we need to update /metadata call to show ownership info so I can pre-fill those